src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, Color color) {

```

Comment:
You should call the superclass constructor super(color) to set the color of the figure correctly.

Reply:
I am not sure if I did it correctly and I sure that I do not know the purpose of it ;)